### ISSUE TITLE
any suggestions on how to label_image on batch images not one by one?

### ISSUE THREAD

**Issue by fengdragon** [googlecodelabs/tensorflow-for-poets-2#24]
thanks for the great work and helpful code, I'm a newer on TF,
and I will appreciate your help with this situation:

I have followed the tutorial to retrain my own model, using my own train datas.
and got the saved model *.pb file,
then when I want to label another whole dataset images,
it was so slow to classify the images one by one,
and is there a way to feed a batch size images to classification progress?
as I searched.. it seems to me that there are some ops not supporting batch,
any suggestions or code referred would be great and helpful, thanks in advance!

**Comment by hcchengithub** [comment]
TensorFlow Tutorial #09
by [Magnus Erik Hvass Pedersen](http://www.hvass-labs.org/) / [GitHub](https://github.com/Hvass-Labs/TensorFlow-Tutorials) / [Videos on YouTube](https://www.youtube.com/playlist?list=PL9Hr9sNUjfsmEu1ZniY0XpHSzl5uihcXZ)

**Comment by reid3290** [comment]
I have the same question and the tutorial video recommended by @hcchengithub doesn't help.
